modules/clightning.nix

```diff
@@ -146,11 +149,12 @@ in {
 
   config = mkIf cfg.enable {
     services.bitcoind = {
-      enable = true;
+      enable = mkForce cfg.useBitcoind;
```

Comment:
Trustedcoin is useful to run CLN with a pruned `bitcoind` (it only falls back to explorers if the local `bitcoind` can't answer a request), so always deactivating `bitcoind` to run `trustedcoin` is suboptimal.

Reply:
My understanding is that trustedcoin indeed loads [from bitcoind if available](https://github.com/nbd-wtf/trustedcoin#using-bitcoind). In that case, I think it makes sense to leave `enable = true;` as is.

Thanks @seberm btw for another attempt at this PR!